### DISCUSSION TITLE
plugin.xml更新，ide中不会有变化

### DISCUSSION THREAD

**Discussion by yuyu1025** [liuzhihang/doc-view#68] [Q&A]
我也想参与到项目的开发里来，但是更改了plugin.xml，尝试了一下重新构建，也没啥效果，希望有大佬可以指导一下~

**Comment by liuzhihang** [comment]
你拉下来代码，等待依赖拉下来之后，在 idea 右边 gradle 可以看到 build 和 run  ，run的时候会在本地拉起一个 plugin.xml 配置对应版本的 idea，在沙箱环境就可以看效果，同事可以在原本的代码中断点调试。

**Comment by liuzhihang** [comment]
可以参考我写的文章：
https://mp.weixin.qq.com/s/Bm8sLyWWiNSlP3l7gz6h_g

https://mp.weixin.qq.com/s/75h_j5McIRQ3Z0fB72MZuA

里面部分可能不符合现在的版本了，可以参考官网
https://plugins.jetbrains.com/docs/intellij/welcome.html?from=jetbrains.org#contribute

**Reply by yuyu1025** [reply]
我刚刚找到了~不在plugin里，在配置类里~~~我人傻了

**Reply by yuyu1025** [reply]
感谢大佬回复
